crm_claim_type/i18n/es.po

```diff
@@ -0,0 +1,90 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* crm_claim_type
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-08-06 16:36+0000\n"
+"PO-Revision-Date: 2015-08-06 16:36+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: crm_claim_type
+#: field:crm.claim.type,active:0
+msgid "Active"
+msgstr "Activo"
+
+#. module: crm_claim_type
+#: model:ir.model,name:crm_claim_type.model_crm_claim
+msgid "Claim"
+msgstr "Reclamo"
```

Comment:
@osvalr can you change Reclamación here, and put on es_XX Reclamo here in latinamérica Reclamación is not corret but it looks like in Spain it is.